sandbox/deploy.sh

```diff
@@ -10,8 +10,16 @@ if [ "$1"x = "x" ]; then
     ns_num=1;
 elif [ "$1x" == "raftx" ]; then
     ns_num=3
-elif [ "$1x" == "master_slave" ]; then
+elif [ "$1" == "master_slave" ]; then
```

Comment:
If possible, please solve this pull request.  I am not familiar with git.  And I have made a mistake, which creates the pull request from my `master` branch. It is really annoying.